service/accounting/model/service.go

```diff
@@ -0,0 +1,99 @@
+	IsDeleted         int32   `json:"is_deleted" validate:"required"`
+}
+
+type UpsertCompanyFiscalYearRequestResponse struct {
+	CompanyId  string `json:"company_id" validate:"required"`
+	StartMonth int32  `json:"start_month" validate:"required"`
+	StartYear  int32  `json:"start_year" validate:"required"`
+	EndMonth   int32  `json:"end_month" validate:"required"`
+	EndYear    int32  `json:"end_year" validate:"required"`
+}
+
+type SetCompanyFirstBankAccountRequest struct {
+	CompanyId         string `json:"company_id" validate:"required"`
+	BranchId          string `json:"branch_id" validate:"required"`
+	BankName          string `json:"bank_name" validate:"required"`
+	BankAccountNumber string `json:"bank_account_number" validate:"required"`
+	BankCode          string `json:"bank_code" validate:"required"`
+}
+
+type SetCompanyFirstCashAccountRequest struct {
+	CompanyId   string `json:"company_id" validate:"required"`
+	BranchId    string `json:"branch_id" validate:"required"`
+	AccountName string `json:"account_name" validate:"required"`
+}
+
+type UpsertCompanyChartOfAccountResponse struct {
+	ChartOfAccount
+}
+
+type GetAccountingBanksRequest struct {
+	Type string `json:"type" validate:"required"`
+}
+
+type GetAccountingBanksResponse struct {
+	Banks []string `json:"banks"`
+}
+
+type GetAccountingChartOfAccountTypesResponse struct {
+	ChartOfAccountTypes []string `json:"types"`
+}
+
+type CompanyChartOfAccountsRequest struct {
+	CompanyId string `json:"company_id" validate:"required"`
+	Keyword   string `json:"keyword"`
+}
+
+type AddCompanyChartOfAccountRequest struct {
+	CompanyId         string  `json:"company_id" validate:"required"`
+	BranchId          string  `json:"branch_id" validate:"required"`
+	AccountCode       string  `json:"account_code" validate:"required"`
+	AccountName       string  `json:"account_name" validate:"required"`
+	AccountGroup      string  `json:"account_group" validate:"required"`
+	BankName          string  `json:"bank_name" validate:"required"`
+	BankAccountNumber string  `json:"bank_account_number" validate:"required"`
+	BankCode          string  `json:"bank_code" validate:"required"`
+	OpeningBalance    float64 `json:"opening_balance" validate:"required"`
+}
+
+type UpdateCompanyChartOfAccountRequest struct {
+	ChartOfAccountId  string  `json:"chart_of_account_id" validate:"required"`
+	CompanyId         string  `json:"company_id" validate:"required"`
+	BranchId          string  `json:"branch_id" validate:"required"`
+	AccountCode       string  `json:"account_code" validate:"required"`
+	AccountName       string  `json:"account_name" validate:"required"`
+	AccountGroup      string  `json:"account_group" validate:"required"`
+	BankName          string  `json:"bank_name" validate:"required"`
+	BankAccountNumber string  `json:"bank_account_number" validate:"required"`
+	BankCode          string  `json:"bank_code" validate:"required"`
+	OpeningBalance    float64 `json:"opening_balance" validate:"required"`
```

Comment:
ini bakal dipake buat perhitungan uang? kalau iya jangan pake flot64

- https://stackoverflow.com/questions/15726535/which-datatype-should-be-used-for-currency (database)
- https://github.com/Rhymond/go-money (go)